src/mame/netronics/exp85.cpp

```diff
@@ -201,6 +239,41 @@ void exp85_state::sod_w(int state)
 	}
 }
 
+/* Memory mapping mgmt during reset and interrupt */
+
+void exp85_state::status_out(u8 status)
+{
+	// In the real hardware this is monitored via the IO/M, S0, S1 
+	// and ALE output pins of the 8085.
+	// Since these pins are not emulated, then we must explicitly get the internal 
+	// interrupt acknowledge state (0x23 or 0x26) and behave as the monitor expects.
+	auto current_pc = m_maincpu->pc();
+	if (status == 0x23 || status == 0x26)
+	{
+		// When an interrupt is triggered the low memory shall be set to the ROM mirror
+		m_is_preparing_interrupt_call = true;
+		m_low_memory_view.select(LOW_MEMORY_ROM_MIRROR_ENTRY);
+	} 
+	else if (m_is_preparing_interrupt_call && (current_pc & 0xff00) == 0x0000)
+	{
+		// Avoids setting the lower memory back to RAM until
+		// it branches to the interrupt handler.
+		m_is_preparing_interrupt_call = false;
+	} 
+	else if (!m_is_preparing_interrupt_call && (current_pc & 0xf000) == 0xf000)
+	{
+		// When the interrupt handler is executing and the address is >= 0xf000
+		// the low memory is mapped to RAM
+		m_low_memory_view.select(LOW_MEMORY_RAM_ENTRY);
+	} 
+		
+}
```

Comment:
@ajrhacker are you happy with this as a temporary solution with the current state of the 8086 CPU device, or would you suggest another way of achieving this?